app/web/src/newhotness/Component.vue

```diff
@@ -6,10 +6,10 @@
     <VButton
       class="border-0 mr-2em"
       icon="arrow--left"
+      label="Back"
```

Comment:
Did any newhotness files actually need to change? I think there's only formatter sorting changes here. Probably best not to commit them for now, to avoid unnecessary conflicts.

Reply:
I'm conflicted about this because all the changes were made by the buck2 linter target - but I agree. I'll remove the changes to files that weren't functionally edited

Reply:
oh, that's totally fine then imo! I wasn't sure if it was official linting or just something your editor did because you were in there.